changelog/internal/metadata.json

```diff
@@ -0,0 +1 @@
+{"runDate": "2024-08-22", "runId": "20240822084923", "specRevision": "0fdbababd77c6fefa307f190de1ae3c99d4eb61f", "specRevisionShort": "0fdbababd77", "versions": ["2023-01-01", "2023-02-01", "2023-10-01", "2023-11-15", "2024-05-30", "2024-08-05"], "migration": 1}
```

Comment:
is this for dev? should we be showing 2025-01-01?

Reply:
no, it is in the main branch so it is prod